libs/shared/decorators/src/lib/utils.ts

```diff
@@ -89,12 +106,7 @@ export const handleException = <T>({
     })
   }
 
-  logger.error(`Error in ${category}.${method}`, {
-    ...info,
-    category,
-    method,
-    error,
-  })
+  logger.error(`Unhandled error in ${category}.${method}: ${error.message}!`)
```

Comment:
Only thing we kow about this error now is that it's not an `Error`, `ValidationError` or a `DatabaseError` so trying to access the `message` property seems unsafe.